core/src/main/scala/org/apache/spark/rdd/RDDCheckpointData.scala

```diff
@@ -24,9 +24,7 @@ import org.apache.spark.Partition
 /**
  * Enumeration to manage state transitions of an RDD through checkpointing
  *
- * {{{
- * [ Initialized --> checkpointing in progress --> checkpointed ]
- * }}}
+ * [ Initialized --{@literal >} checkpointing in progress --{@literal >} checkpointed ]
```

Comment:
- Java
  ![2016-11-28 3 41 11](https://cloud.githubusercontent.com/assets/6477701/20658544/1c962f72-b581-11e6-9126-1b0a6fc8354a.png)

Scaladoc not found.